frontend/src/pages/ReviewDashboardPage/index.tsx

```diff
@@ -0,0 +1,70 @@
+import { useNavigate } from 'react-router';
+
+import DashboardIcon from '@/assets/dashboard.svg';
+import { Button } from '@/components';
+
+// TODO: 상수명 단수로 고치기
+import { ROUTES } from '@/constants/routes';
+import useModals from '@/hooks/useModals';
+
+import PasswordModal from './components/PasswordModal';
+import * as S from './styles';
+
+interface ReviewDashboardPageProps {
```

Comment:
리뷰방이 생성되면 복사하는 url이기존의 리뷰 작성 페이지에서 리뷰 연결페이지 (=ReviewDashboardPage)로 변경 될거에요.
그렇다면 ReviewDashboardPage 안에서  `리뷰 그룹 정보 조회` 요청을 통해 리뷰이, 프로젝트명을 받아와서 사용할 거고, ErrorBoundary, Suspense를 적용한다면 현재 ReviewDashboard안의 있는 코드들은 리뷰 상세정보 페이지등과 같이 다른 컴포넌트로 빼고 ReivewDashboardPage안에서 불러와서 사용해야될 것 같네요.

```suggestion
const ReviewDashboardPage () =>{
return (
<ErrorSuspenseContainer>
 <RevuewDasgbord/>
 </ErrorSuspenseContainer>
)
}
```
위의 구조처럼 한다면, ReviewDashboard 컴포넌트안에서  `리뷰 그룹 정보 조회` 요청을 할거고, 요청 하는 동안에 suspense의 fallback이 오류 시에는 errorboundary의 fallback이 작동하겠네요.

Reply:
오 이건 전혀 생각하지 못했네요 😮
URL이 보여주는 페이지가 달라진만큼 그에 따른 새로운 Suspense와 에러 바운더리도 챙겼어야 했는데... 몰랐읍니다
바다의 제안대로 고칠 것 같은데 페이지 이름들을 어떻게 하면 좋을지 고민이네요 🤪

Reply:
suspense 및 error boundary 처리를 위한 Wrapper 컴포넌트를 새로 만드는 것보다는 라우터에서 `ReviewDashboard `컴포넌트를 `ErrorSuspenseContainer`로 감싸는 게 나을 것 같다는 생각이 들어서 라우터 페이지에서 감싸주는 방식으로 고쳐봤습니당